style.css

```diff
@@ -0,0 +1,98 @@
+body {
+  text-align: center;
+  background-color: rgb(116, 86, 86);
+  font-family: Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
+}
+
```

Comment:
It looks great! The only thing I would've done, (which I actually didn't on mine, but should have) is adding cursor: pointer to the image/button itself. It's a better user experience to see clickable things with the cursor pointer.